app/src/components/Wallet.js

```diff
@@ -23,53 +24,90 @@ function Wallet({ isLoading }) {
 
   return (
     <Box padding={0}>
-      <div
-        css={`
-          display: flex;
-          align-items: center;
-          padding: ${3 * GU}px;
-          border-bottom: 1px solid ${theme.border};
-        `}
-      >
-        <EthIdenticon
-          address={account}
-          radius={100}
-          scale={1.7}
-          css={`
-            margin-right: ${1.5 * GU}px;
-          `}
-        />
-        <span
-          css={`
-            ${textStyle('title4')}
-          `}
-        >
-          {shortenAddress(account, 4)}
-        </span>
-      </div>
-
-      {/** TODO we need more validations to know what to show in this part of the wallet like if the user was already registered or not */}
-      {!addressExist && account && (
+      {!account ? (
         <div
           css={`
+            display: flex;
             padding: ${3 * GU}px;
+            flex-direction: column;
             text-align: center;
+            align-items: center;
           `}
         >
-          <h5
+          <div
+            css={`
+              border-radius: 50%;
+              border: 2.3px solid #c2c2c2;
+              width: ${7 * GU}px;
+              height: ${7 * GU}px;
+            `}
+          >
+            <IconConnect
+              css={`
+                color: #c2c2c2;
+              `}
+              size="large"
+            />
+          </div>
+          <span
             css={`
               ${textStyle('body1')};
-              color: ${theme.surfaceContent};
-              margin-bottom: ${3 * GU}px;
+              margin-top: ${2 * GU}px;
             `}
           >
-            Connect with BrightID
-          </h5>
-          <QRCode
-            value={deepLink}
-            style={{ width: `${17 * GU}px`, height: `${17 * GU}px` }}
-          />
+            To start please connect your account
+          </span>
         </div>
+      ) : (
+        <>
+          <div
+            css={`
+              display: flex;
+              align-items: center;
+              padding: ${3 * GU}px;
+              border-bottom: 1px solid ${theme.border};
```

Comment:
Maybe we could center it also for small screen sizes.

![Screen Shot 2020-07-15 at 10 16 55 AM](https://user-images.githubusercontent.com/22663232/87549628-6e7eae00-c684-11ea-85b0-cf1bc6e64b4c.png)

```suggestion
              border-bottom: 1px solid ${theme.border};
              justify-content: center;
```